GetDKAN/dkan
# Datastore and individual dataset page API doc improvements

## QA Steps
Open [QA site](https://dkan-qa-3625.ci.civicactions.net/) /api and check
- the new {datasetId}/{index} endpoints are properly documented
- The search endpoints are properly documented

Open a dataset page, click to the API docs for that dataset, and check
- The datastore query docs are clearly and properly documented
